src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+	Tag tag;
+	Person person;
+	boolean isAdded;// 1 means the tag is added, o means the tag is removed
+
+	public Tagging(Person person, Tag tag, boolean k) {
+		this.person = person;
```

Comment:
i think you are using tabs for indentation; please set Eclipse to replace tabs with spaces.